src/mol-gl/webgl/render-item.ts

```diff
@@ -218,6 +218,8 @@ export function createRenderItem<T extends string>(ctx: WebGLContext, drawMode:
             if (drawCount === 0 || instanceCount === 0) return;
 
             const program = programs[variant];
+            if (!program.isReady()) return;
```

Comment:
I don't think this will work together with the animation generation. 

I've pushed https://github.com/molstar/molstar/pull/1655/commits/bdd1805620f0e524874c79deaa534d88a530a268 with a suggestion to move this into the commit queue -- although that still has some drawbacks and would have to solve the init of the post-processing shaders separately.

Alternatively, we can update the renderable "lifecycle" to support splitting the compilation of shaders into a separate step?